flake.nix

```diff
@@ -18,9 +18,17 @@
       owner = "NixOS";
       repo = "nixpkgs";
     };
+    cardano-node-repo.url = "github:input-output-hk/cardano-node/1.35.0";
+    cardano-db-sync.url = "github:input-output-hk/cardano-db-sync/13.0.0";
+    #cardano-private-testnet-setup = {
```

Comment:
Oh, I just noticed this is commented out, so it seems you tried this first (see my other comment about avoiding `pkgs.fetchFromGitHub`). You _should_ use a flake input for this, you just need to add `flake = false;`